site/en/docs/webstore/best_practices/index.md

```diff
@@ -150,6 +150,47 @@ Social & Communications
   * Travel
   * Well-being
 
+### Choose your theme's category well {: #choose-theme-category-well}
+
+The developer console also asks you specify a category for your theme. Choose the most appropriate category:
+
+Artist Series
+: Themes in collections curated by Google.
+
+Animals
+: Themes featuring your favorite animals.
+
+Art & Design
+: Themes build merely to make your browser look pretty.
+
+Cars
+: Themes featuring current and classic cars.
```

Comment:
Did these come from some official, already-public definitions or is there scope to change them? Some of them feel a bit weird, e.g with this one - what if I want to feature future cars? Also what if I'm just using the color theme of a car brand but not actually featuring a specific car? They seem too limiting, and I think something like "Themes relating to cars, such as classic or current cars" would be better.

Reply:
Got it. I left some suggestions :)